docs/BR.md

```diff
@@ -1493,124 +1498,124 @@ Certificates MUST be of type X.509 v3.
+   This extension MUST be present and MUST NOT be marked critical. It MUST contain the HTTP URL of the CA's CRL service.
 
-d. basicConstraints
+c. `authorityInformationAccess`
 
-> This extension MUST be present and MUST be marked critical. The cA field MUST be set true. The pathLenConstraint field MAY be present.
+   With the exception of stapling, which is noted below, this extension MUST be present. It MUST NOT be marked critical, and it MUST contain the HTTP URL of the Issuing CA's OCSP responder (`accessMethod` = 1.3.6.1.5.5.7.48.1). It SHOULD also contain the HTTP URL of the Issuing CA's certificate (`accessMethod` = 1.3.6.1.5.5.7.48.2).
 
-e. keyUsage
+   The HTTP URL of the Issuing CA's OCSP responder MAY be omitted, provided that the Subscriber "staples" the OCSP response for the Certificate in its TLS handshakes [RFC4366].
 
-This extension MUST be present and MUST be marked critical. Bit positions for keyCertSign and cRLSign MUST be set. If the Subordinate CA Private Key is used for signing OCSP responses, then the digitalSignature bit MUST be set.
+d. `basicConstraints`
 
-f. nameConstraints (optional)
+   This extension MUST be present and MUST be marked critical. The cA field MUST be set true. The pathLenConstraint field MAY be present.
 
-> If present, this extension SHOULD be marked critical\*.
->
-> \* Non-critical Name Constraints are an exception to RFC 5280 (4.2.1.10), however, they MAY be used until the Name Constraints extension is supported by Application Software Suppliers whose software is used by a substantial portion of Relying Parties worldwide.
+e. `keyUsage`
 
-g. extkeyUsage (optional)
+   This extension MUST be present and MUST be marked critical. Bit positions for `keyCertSign` and `cRLSign` MUST be set. If the Subordinate CA Private Key is used for signing OCSP responses, then the `digitalSignature` bit MUST be set.
 
-> For Subordinate CA Certificates to be Technically constrained in line with section 7.1.5, then either the value id-kp-serverAuth [RFC5280] or id-kp-clientAuth [RFC5280] or both values MUST be present\*\*.
->
-> Other values MAY be present.
->
-> If present, this extension SHOULD be marked non-critical.
->
-> \*\* Generally Extended Key Usage will only appear within end entity certificates (as highlighted in RFC 5280 (4.2.1.12)), however, Subordinate CAs MAY include the extension to further protect relying parties until the use of the extension is consistent between Application Software Suppliers whose software is used by a substantial portion of Relying Parties worldwide.
+f. `nameConstraints` (optional)
 
+   If present, this extension SHOULD be marked critical[^*].
+
+[^*]: Non-critical Name Constraints are an exception to RFC 5280 (4.2.1.10), however, they MAY be used until the Name Constraints extension is supported by Application Software Suppliers whose software is used by a substantial portion of Relying Parties worldwide.
```

Comment:
Unclear what syntax this is trying to use, but you don't seem to have used the same syntax in 6.1.5 (thankfully(?) the browser cleanup proposal will significantly rewrite and simplify that section?)

Reply:
The `[^*]` blob creates an auto-numbered footnote (https://pandoc.org/MANUAL.html#footnotes). Not actually sure why I used it there and didn't in 6.1.5—it's possible that since 6.1.5 contained multi-asterisk footnotes, I was worried about modifying the doc too much. I'll wait to see how the cleanup ballot modifies those sections and then we can revisit.